spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplate.java

```diff
@@ -799,6 +783,43 @@ private DatastoreReaderWriter getDatastoreReadWriter() {
 		return builder.build();
 	}
 
+	private <T> Value<?> getValue(Example<T> example, FullEntity<IncompleteKey> probeEntity,
+					DatastorePersistentEntity<?> persistentEntity, DatastorePersistentProperty persistentProperty) {
+		Value<?> value;
+		if (persistentProperty.isIdProperty()) {
+			value = getIdValue(example, persistentEntity, persistentProperty);
+		}
+		else {
+			value = probeEntity.getValue(persistentProperty.getFieldName());
+		}
+		return value;
+	}
+
+	private <T> boolean notIgnoredProperty(Example<T> example, DatastorePersistentProperty persistentProperty) {
```

Comment:
It may be easier to read as `isIgnoredProperty()`, removing the negation here and putting the negation in `getDatastoreReadWriter()` method instead. Our brains are very used to the `isBlah` pattern.